openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -798,6 +799,49 @@ Here is another example JWT not only proving possession of a private key but als
 }
 ```
 
+#### `ldp_vp_2.0` Key Proof Type {#ldp_vp-proof-type}
+
+The verifiable presentation as defined by [@VC_DATA_2.0] signed using Data Integrity that are used as Key Proofs MUST contain the following elements:
```

Comment:
Is Data Integrity a defined term? If yes, please capitalize this in the entire PR/Document

Reply:
Was just one more occurrence

https://github.com/openid/OpenID4VCI/pull/66/commits/93c20c80e547ce70b655ca15d38477b7e7f87200